bundles/org.openhab.binding.senechome/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -122,6 +113,11 @@
 				<description>Duration of stable values until state is changed, defined in seconds</description>
 				<default>120</default>
 			</parameter>
+			<parameter name="useHttp" type="boolean" required="false">
```

Comment:
Please update the i18n property file to cover this new parameter.

Reply:
updated